tools/src/main/java/org/apache/rocketmq/tools/command/broker/GetBrokerConfigCommand.java

```diff
@@ -121,8 +124,8 @@ protected void getAndPrint(final MQAdminExt defaultMQAdminExt, final String prin
             return;
         }
 
-        for (Object key : properties.keySet()) {
-            System.out.printf("%-50s=  %s\n", key, properties.get(key));
+        for (Entry<Object, Object> keyEntry : properties.entrySet()) {
+            System.out.printf("%-50s=  %s\n", keyEntry.getKey(), keyEntry.getValue());
```

Comment:
keyEntry could be renamed to entry, and so is the obove

Reply:
already edited